### PR TITLE
Feature/release 1 3 5

### PR DESCRIPTION
**Pull request by ehealy**
@gerbercj 
cc @Tapjoy/eng-group-ops 

Rebuilt after cleaning up master

### CODE REVIEW

File: templates/sh.erb [hunk 1]
```diff
@@ -211,7 +212,8 @@ function create_symlinks(){
     [ -n "$USE_FLAT_RELEASE_DIRECTORY" ] && return
 
     export CURRENT_DIR="$INSTALL_ROOT/current"
-    if [ -e "$CURRENT_DIR" ] ; then
+    if [ -L "$CURRENT_DIR" ] ; then
```
Comment (gerbercj) [inline templates/sh.erb]: I was thinking that we settled on `if [ -e "$CURRENT_DIR" ] || [ -h "$CURRENT_DIR" ] ; then`

Comment (ehealy) [inline templates/sh.erb]: Crap. We did. Damage from all the branch shuffling. Fixing

**Comment by ehealy** [comment]
@gerbercj fixed merge damage.

**Comment by gerbercj** [comment]
LGTM

**Pull request merged**